src/main/views/courts/courts.njk

```diff
@@ -44,10 +41,37 @@
 
     <div id='numberOfCourts' class='govuk-hint'></div>
 
+    {% set courtItems = [] %}
+    {% for court in courts|sort(false, true, 'name') %}
+      {% set courtItems = (courtItems.push([
+        {
+          'attributes': {
+            'type': 'name',
+            'value': court.name,
+            'name': court.slug
+          },
+          'text': court.name
+        },
+        {
+          'attributes': {'type': 'displayed', 'value': court.displayed},
+          'text': '' if court.displayed else 'closed'
+        },
+        {
+         'attributes': {'type': 'updated_at', 'value': court.updated_at},
```

Comment:
Do we need the value attributes? If they're the same value as the 'td' then we can just get the value from there.